StackBlur/.cxx/abi_configuration_4c11233e.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This review adds a line to the .gitignore which would seem to want to ignore this file (and several others in this review.

Reply:
Oh yeah, they need to be git rm-ed, thanks for the heads up. I am cleaning a bit more the repo.